enterprise-suite/templates/prometheus.yaml

```diff
@@ -139,6 +139,11 @@ spec:
             - --defaultMonitorsFile=/etc/es-monitor-api/default-monitors.json
             - --prometheusTemplate=/etc/es-monitor-api/prometheus.yml
             - --prometheusDomain={{ .Values.prometheusDomain }}
+            {{- if .Values.alertmanagerAddress}}
+            - --alertmanagerAddress={{ .Values.alertManagerAddress }}:{{ .Values.alertManagerPort }}
+            {{ else }}
+            - --alertmanagerAddress=alertmanager:{{ .Values.alertManagerPort }}
+            {{ end }}
```

Comment:
Wouldn't it be better to have a flag "createAlertManager" that we can use as the gate for creating the internal alertmanager instead?

Then you can just set `alertManagerAddress` to the default value `alertmanager`, and we can get rid of the conditional logic that depends on whether the address is nil or not.

I think this is also a bit more explicit/clearer to users. It's not as obvious that setting the address will cause the internal alertmanager to not be created.

Reply:
Changed it to `createAlertManager` + `alertManagers` for addresses.